docs/reports/reports.rst

```diff
@@ -3,4 +3,226 @@
+
+* **Calculated columns** - Select the function that you want to apply to individual columns. Calculated columns display count, average, sum, or the minimum or maximum values from a selected field. They are only available when a grouping is used to show a calculation for that grouping. Continuing with the previous example of grouping by a contact ID number or email address, a COUNT calculation displays how many times that contact record appears on the report if not for the grouping.
+
+Graphs
+======
+
+.. image:: images/mautic-report-graph.png
+   :align: center
+   :alt: Screenshots of Mautic Report Data
+
+|
+
+Some report types display graphs for visualization purposes. You can include such graphs in reports and use them in dashboard widgets.
+
+* To select an available graph and add it to the report, click the name of the graph to move it from the left column to the right.
+* To remove a graph from a report, click the name in the right column to move it to the left.
+  
+The availability and types of graphs vary by report type. 
+
+Schedule
+========
+
+Mautic enables scheduling Emails to send downloadable links containing the report data in the ``.csv`` file format.
+
+Use the toggle switch to enable or disable sending Reports via Email.
+
+* Email report - Set this toggle bar to **Yes** to see additional options.
+
+* To - Specify the Email addresses that should receive the report. To send to multiple recipients, separate their Email addresses with a comma. For example, ``example1@yourcompany.com``, ``example2@yourcompany.com``. 
+
+* **Every** - Select the frequency with which you’d like to automatically send the report:
+
+  - **now** - Sends the report once, when the report is saved.
+  - **day** - Sends the report every day at midnight in your time zone.
+  - **week** - After selecting week, select the day of the week you’d like to send the report. The report is sent at midnight in your time zone each week on the selected day.
+  - **month** - After selecting month, select either first or last and a day of the week. For example, set your report to be automatically sent on the first Monday or last Friday of each month. 
+
+Alternatively, you can select Weekdays to send the report on the first or last weekday of each month.
+
+Once you’ve set all of the options you’d like in the **Details**, **Data**, **Graphs**, and **Schedule** tabs, click **Save & Close** to save the report. Clicking **Apply** saves the progress you’ve made on building the report and keeps you in the edit mode.
+
+Cron job to schedule reports
```

Comment:
```suggestion
.. vale off
Cron job to schedule Reports
.. vale on
```

This is a valid use of vale off/on - we need to capitalise Reports as we are referring to a 'thing' in Mautic, but it is not allowed due to our rule that says only the first letter of the heading should use a capital letter. So we tell Vale we explicitly want it to ignore our rules because we know that we are correct in breaking them.